app/gosns/create_topic.go

```diff
@@ -0,0 +1,47 @@
+package gosns
+
+import (
+	"net/http"
+
+	"github.com/Admiral-Piett/goaws/app"
+	"github.com/Admiral-Piett/goaws/app/common"
+	"github.com/Admiral-Piett/goaws/app/interfaces"
+	"github.com/Admiral-Piett/goaws/app/models"
+	"github.com/Admiral-Piett/goaws/app/utils"
+	log "github.com/sirupsen/logrus"
+)
+
+func CreateTopicV1(req *http.Request) (int, interfaces.AbstractResponseBody) {
+	requestBody := models.NewCreateTopicRequest()
+	ok := utils.REQUEST_TRANSFORMER(requestBody, req, false)
+	if !ok {
+		log.Error("Invalid Request - CreateTopicV1")
+		return createErrorResponseV1(ErrInvalidParameterValue.Type)
+	}
+
+	topicName := requestBody.Name
+	topicArn := ""
+	if _, ok := app.SyncTopics.Topics[topicName]; ok {
+		topicArn = app.SyncTopics.Topics[topicName].Arn
+	} else {
+		topicArn = "arn:aws:sns:" + app.CurrentEnvironment.Region + ":" + app.CurrentEnvironment.AccountID + ":" + topicName
+
+		log.Debug("Creating Topic:", topicName)
```

Comment:
I might put this as `Info` - it's a good thing to see if you're looking for validation that `GoAWS` is doing what you think it should.